pkg/console/operator/health_deployment.go

```diff
@@ -0,0 +1,41 @@
+package operator
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	deploymentv1 "k8s.io/api/apps/v1"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+
+	"github.com/openshift/console-operator/pkg/console/status"
+	deploymentsub "github.com/openshift/console-operator/pkg/console/subresource/deployment"
+)
+
```

Comment:
separated this out so that helpers for each check can be co-located with the status they support.